sys-process/bashtop/bashtop-0.9.23.ebuild

```
@@ -0,0 +1,31 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Resource monitor that shows usage and stats"
+HOMEPAGE="https://github.com/aristocratos/bashtop"
+SRC_URI="https://github.com/aristocratos/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+inherit eutils
```

Usually inherits go under a blank line, under EAPI:
```suggestion
EAPI=7

inherit eutils
```

---

I have made the changes accordingly